expression/integration_test.go

```diff
@@ -3507,6 +3507,25 @@ func (s *testIntegrationSuite) TestArithmeticBuiltin(c *C) {
 	c.Assert(terror.ErrorEqual(err, types.ErrOverflow), IsTrue)
 	c.Assert(rs.Close(), IsNil)
 
+	//IntDiv overflow test
```

Comment:
@Tjianke 
Cover only two of four overflow cases here looks a little strange to me.

In my previous comment I suggested to add test cases for "Signed-Unsigned" & "Unsigned-Signed" only because the other two (UU and SS) exists in `(*testEvaluatorSuite) TestArithmeticDivide()`.

I think place `US` and `SU` in `(*testEvaluatorSuite) TestArithmeticDivide()` would be easier, and would be better.

Reply:
while adding tests in `TestArithmeticDivide`, I found out another issue #22759, root cause is lying [here](https://github.com/pingcap/tidb/blob/master/types/overflow.go#L203)